tests/pytest_tests/unit_tests_old/test_cli.py

```diff
@@ -251,8 +251,8 @@ def test_sweep_scheduler(runner, mock_server, test_settings):
         assert sweep_id == "test"
         assert (
             runner.invoke(
-                cli.sweep,
```

Comment:
you basically removing sweep tests, maybe a better option is to have it test both code paths?